requirements/frozen.yml

```diff
@@ -129,6 +129,7 @@ dependencies:
     - bravado-core==5.17.0
     - colorama==0.4.4
     - cycler==0.10.0
+    - decord==0.6.0
```

Comment:
Just FYI, when I installed decord in this env, it didn't install any additional transitive dependency. That's why I don't add any other dependency here.